plotgui.py

```diff
@@ -330,6 +405,49 @@ def setPixmap(self, w, h):
 
         self.draw()
 
+    def updateColorBarScale(self):
+        self.setPixmap()
+
+    def updateDataIndicatorValue(self, y_val):
+        cv = self.model.currentView
+
+        if cv.colorby not in _MODEL_PROPERTIES or \
+           not cv.data_indicator_enabled[cv.colorby]:
+            return
+
+        if self.data_indicator:
+            data = self.data_indicator.get_data()
+            # use norm to get axis value if log scale
+            if cv.color_scale_log[cv.colorby]:
+                y_val = self.image.norm(y_val)
+            self.data_indicator.set_data([data[0], [y_val, y_val]])
+            dl_color = invert_rgb(self.colorbar.get_cmap()(y_val), True)
+            self.data_indicator.set_c(dl_color)
+            self.draw()
+
+    def updateDataIndicatorVisibility(self):
+        av = self.model.activeView
+        if self.data_indicator and av.colorby in _MODEL_PROPERTIES:
+            val = av.data_indicator_enabled[av.colorby]
+            self.data_indicator.set_visible(val)
+            self.draw()
+
+    def updateColorMap(self, colormap_name, property_type):
+        if self.colorbar and property_type == self.model.activeView.colorby:
+            self.colorbar.set_cmap(colormap_name)
+            self.image.set_cmap(colormap_name)
+            self.colorbar.draw_all()
+            self.draw()
+
+    def updateColorMinMax(self, property_type):
+        av = self.model.activeView
+        if self.colorbar and property_type == av.colorby:
+            clim = av.getColorLimits(property_type)
+            self.colorbar.set_clim(*clim)
+            self.data_indicator.set_data((clim[0], clim[1]),
```

Comment:
Would `clim[:2]` work instead of `(clim[0], clim[1])`?

Reply:
Yup. Updated.